pkg/controller/user/secret.go

```diff
@@ -34,10 +37,11 @@ func (r *ReconcileUser) newSecretForUser(ctx context.Context, user *registriesv1
 
 	err := r.client.Get(ctx, types.NamespacedName{Name: sec.Name, Namespace: sec.Namespace}, sec)
 	if errors.IsNotFound(err) {
-		pw, err := helper.NewRandomPassword(16)
+		pw, err := helper.NewRandomPassword(user.Spec.PasswordStrength)
```

Comment:
Implement some default value? In case `passwordStrength` is omitted, or set to a nonsencical value?

Reply:
:+1: 
Also, adding kubebuilder flags for a minimum / maximum value to the `PasswordStrength` field of the `UserSpec` struct might be sensible: 
```
type UserSpec struct {
	[...]
	// The effective length of the generated user password
	// +kubebuilder:validation:Minimum=8
	// +kubebuilder:validation:Maximum=128
	PasswordStrength int32 `json:"passwordStrength"`
[...]
```